src/main/webapp/app/pages/LevelOfEvidencePage.tsx

```diff
@@ -188,150 +197,220 @@ export default class LevelOfEvidencePage extends React.Component<
-              <h4>{LEVEL_TITLE[this.version]}</h4>
-              <div>
-                <span
-                  onClick={() =>
-                    this.toggleVersion(
-                      V2_RELATED_LEVELS.includes(this.version)
-                        ? Version.V1
-                        : Version.V2
-                    )
-                  }
-                  className={mainStyles.btnLinkText}
-                >
-                  {LEVEL_SUBTITLE[this.version]}
-                </span>
-              </div>
-            </Col>
-          </Row>
-          <Row>
-            <Col className={'d-md-flex justify-content-center mt-2'}>
-              <div
-                style={{
-                  maxWidth: [
-                    Version.AAC,
-                    Version.FDA,
-                    Version.V1,
-                    Version.V2,
-                  ].includes(this.version)
-                    ? undefined
-                    : IMG_MAX_WIDTH,
-                }}
-              >
-                <OptimizedImage
-                  style={{ width: '100%' }}
-                  progressiveLoading
-                  src={`content/images/level_${this.version}.png`}
-                />
-                {this.version === Version.AAC ? (
-                  <div className="text-right">
-                    <span
-                      style={{
-                        marginRight: this.props.windowStore.isLargeScreen
-                          ? '130px'
-                          : '35px',
-                      }}
-                    >
-                      <sup>1</sup>{' '}
-                      <Linkout link="https://www.sciencedirect.com/science/article/pii/S1525157816302239?via%3Dihub">
-                        Li, MM et al., J Mol Diagn 2017
-                      </Linkout>
-                    </span>
-                  </div>
-                ) : null}
-              </div>
-            </Col>
-          </Row>
-        </Tab>
-      );
-    });
+    const activeLinkStyle = {
+      fontWeight: 700,
+    };
+
+    const activeLinkUnderLine = {
+      backgroundColor: 'rgb(9,104,195)',
+      borderRadius: 20,
+      height: 4,
+      width: '50%',
+      margin: 'auto',
+      transtion: 'all 1s',
+      marginTop: 2,
+    };
 
     return (
       <DocumentTitle title={getPageTitle(PAGE_TITLE.LEVELS)}>
         <Row className="justify-content-center">
           <Col lg={10}>
             <div className="levels-of-evidence">
-              <>
-                <Tabs
-                  defaultActiveKey={
-                    [Version.DX, Version.PX, Version.FDA_NGS].includes(
-                      this.version
-                    )
-                      ? Version[this.version]
-                      : Version.V2
-                  }
-                  id="level-type-tabs"
-                  onSelect={k => this.toggleVersion(Version[k || Version.V2])}
+              <Navbar className="mb-3">
+                <Nav
```

Comment:
Why we changed to Navbar from Tabs?

Reply:
The Tabs component does not support routing to other pages. I tried a work around but it proved to be too buggy and not very user friendly, so I decided to change to something that has native support for routing.